main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Editor/TextEditorViewContent.cs

```diff
@@ -87,6 +88,7 @@ protected override async Task<Control> OnGetViewControlAsync (CancellationToken
 
 				await Init (editor, impl);
 				HasUnsavedChanges = impl.IsDirty;
+				await UpdateStyleParent (Owner, editor.MimeType, default);
```

Comment:
```suggestion
				await UpdateStyleParent (Owner, editor.MimeType, editorOptionsUpdateCancellationSource.Token);
```

And also initialize the CancellationTokenSource regardless of update being called.

Reply:
Needs to run on GetView.

Reply:
Well, if we get an update while the view is constructed, we will have conflicting data being set.

Reply:
hm, in any case it's the wrong token OnGetViewControlAsync seems to get a token - wrong sorting of parameters btw.